src/libraries/Microsoft.Extensions.Hosting/src/HostBuilder.cs

```diff
@@ -50,7 +50,8 @@ public IHostBuilder ConfigureHostConfiguration(Action<IConfigurationBuilder> con
         /// <summary>
         /// Sets up the configuration for the remainder of the build process and application. This can be called multiple times and
         /// the results will be additive. The results will be available at <see cref="HostBuilderContext.Configuration"/> for
```

Comment:
For me, one of the problems with the current wording is this sentence:

> The results will be available at <see cref="HostBuilderContext.Configuration"/> for subsequent operations, as well as in <see cref="IHost.Services"/>.

It is a little unclear what "subsequent operations" means here. Does it mean for subsequent `ConfigureAppConfiguration` delegates? Or for operations after the whole AppConfiguration has been built?

Reply:
Possible variants:
* Current from PR
* ```Sets up the configuration for the remainder of the build process and application. This can be called multiple times and the results will be additive. The results will be available after full <see cref="IConfiguration"/> creation at <see cref="HostBuilderContext.Configuration"/> and in <see cref="IHost.Services"/>. ```
* Something else :)

Reply:
I like that middle one.  @maryamariyan @Tratcher - thoughts?

Reply:
How about:
`Sets up the configuration for the remainder of the build process and application. The <see cref="HostBuilderContext.Configuration"/> passed in is the host's configuration built from calls to <see cref="ConfigureHostConfiguration"/>. This can be called multiple times and the results will be additive. After all calls have been processed, <see cref="HostBuilderContext.Configuration"/> will be updated with the results for future build steps. The resulting configuration will also be available in the <see cref="IHost.Services"/> DI Container.`

Reply:
I think @Tratcher 's edition is the clearest of all to follow.